services/fishStockings.service.ts

```diff
@@ -1395,4 +1336,9 @@ export default class FishStockingsService extends moleculer.Service {
       );
     }
   }
+
+  @Method
+  async handleCache() {
+    await this.broker.cacher.clean('public.**');
+  }
```

Comment:
sita dalyka galima igyvendint `@Event` t.y. `public` service pasidaryt `@Event` kurio name `fishStockings.updated` ar `created` ar `deleted` ir pan - ir tada daryt clean'a. Is kitos puses yra `fishStockings.*` - visu eventu klauso. As nesiulyciau cia to daryt - geriau event'ais o ne await'ais - jie trukdo response'u gavima